src/tests.rs

```diff
@@ -206,3 +364,11 @@ fn test_bundle_deserialize() {
     assert_that(&bun.maintainers.unwrap().len()).is_equal_to(&1);
     assert_that(&bun.custom.unwrap().len()).is_equal_to(&2);
 }
+
+
+// Check that a missing file results in an error (not a panic)
+#[test]
+fn test_bundle_from_file_not_found() {
+    let bun = Bundle::from_file("no/such/file.json");
+    assert_that(&bun.is_err()).is_true();
```

Comment:
Well, this panicked:

```
thread 'tests::test_bundle_from_file_not_found' panicked at 'file not found: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

Reply:
@radu-matei can you please retest this?

I rebased, and if I run `cargo test`, all seems to be passing now.